src/content/api/stats.mdx

```diff
@@ -176,7 +177,7 @@ The `errors` and `warnings` properties each contain a list of strings. Each stri
 ``` bash
 ../cases/parsing/browserify/index.js
 Critical dependencies:
-2:114-121 This seem to be a pre-built javascript file. Even while this is possible, it's not recommended. Try to require to original source to get better results.
+2:114-121 This seems to be a pre-built javascript file. Even while this is possible, it's not recommended. Try to require the original source to get better results.
```

Comment:
Please revert, this is example output of an error which is not controlled by the docs website.

Reply:
oh! I hadn't noticed that.

Reply:
@EugeneHlushko I just committed to change it back to seem